### PR TITLE
ignore diff in imagePullSecrets in tekton-ci

### PR DESCRIPTION
**Pull request by Michkov**
With new imagePullSecrets the argocd marked pipeline service account as OutOfSync. This change is ignoring secrets in list which starts 'pipeline-dockercfg'

### CODE REVIEW

File: argo-cd-apps/base/tekton-ci.yaml [hunk 1]
```diff
@@ -14,6 +14,13 @@ spec:
     namespace: tekton-ci
     server: https://kubernetes.default.svc
 
+
+  ignoreDifferences:
+  - group: ""
+    kind: "ServiceAccount"
```
Comment (lcarva) [inline argo-cd-apps/base/tekton-ci.yaml:20]: Should we set `name` here as well since this is only applicable to the `pipeline` service account anyways?

**Review by lcarva — APPROVED** [review]
/lgtm
/approve

**Review by simonbaird — APPROVED** [review]

**Pull request merged**